<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vetanaCampos</class>
 <widget class="QDialog" name="vetanaCampos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnAgregar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Campo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>57</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineaNombre">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboTipo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Char</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>String</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Integer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Double</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>57</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tama√±o:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinnerLength">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>57</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>bytes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Llave Primaria:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioPrimaria">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Si/No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
